live-build/hooks/600-no-debian.binary

```diff
@@ -38,6 +38,11 @@ PREFIX=binary/boot/filesystem.dir
 # fuse removed because  of https://github.com/snapcore/snapd/pull/11600
 (cd $PREFIX
     chroot . dpkg --purge --force-depends apt libapt-inst2.0 libapt-pkg5.0 lsb-release fuse
+    # The removal of fuse triggers a call to update-initramfs and a
+    # new initrd is created, remove it restore the link to the initrd
```

Comment:
where is the remove step of the new initrd?

Reply:
In the `-f` of `ln -sf...`

Reply:
ah, so just the symlink